src/Universum/Monad/Maybe.hs

```diff
@@ -12,12 +13,27 @@ module Universum.Monad.Maybe
        ) where
 
 import Universum.Applicative (Applicative, pass, pure)
+import Universum.Monad.Reexport (fromMaybe)
 import Universum.Monad.Reexport (Maybe (..), Monad (..))
 
 -- $setup
 -- >>> import Universum.Bool (Bool (..))
 -- >>> import Universum.Function (($))
 -- >>> import Universum.Print (putTextLn)
+-- >>> import Universum.String (readMaybe)
+
+{- | Similar to 'fromMaybe' but with flipped arguments.
+
+>>> readMaybe "True" ?: False
+True
+
+>>> readMaybe "Tru" ?: False
+False
+
+-}
+(?:) :: Maybe a -> a -> a
```

Comment:
Operator without priority? :thinking:

Reply:
@Martoon-00 Oops. Missed this. Well, it's minor. And can be fixed fast and easy.